apps/daffio/src/app/core/header/header.module.ts

```diff
@@ -0,0 +1,31 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+
+import { DaffNavbarModule, DaffButtonModule, DaffContainerModule } from '@daffodil/design';
+import { DaffioLogoModule } from '../logo/logo.module';
+import { DaffioHeaderComponent } from './component/header.component';
+import { DaffioSidebarModule } from '../sidebar/sidebar.module';
+import { DaffioHeaderContainer } from './containers/header.component';
+import { DaffioHeaderItemComponent } from './header-item/header-item.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    RouterModule,
+    DaffNavbarModule,
+    DaffioLogoModule,
+    DaffButtonModule,
+    DaffioSidebarModule,
```

Comment:
Do we need SidebarModule in here?

Reply:
I get an error in the console if I remove it.

Reply:
Removed in 75f81f7